CollAction/Controllers/ProjectsController.cs

```diff
@@ -161,9 +161,15 @@ public async Task<IActionResult> Create(CreateProjectViewModel model)
             foreach (var admin in administrators)
                 _emailSender.SendEmail(admin.Email, subject, confirmationEmailAdmin);
 
-            return View("ThankYouCreate", new ThankYouCreateProjectViewModel()
+            return LocalRedirect($"~/Projects/Create/{Uri.EscapeDataString(project.Name.NormalizeUriPart())}/{project.Id}/thankyou");
+        }
+
+        [Authorize]
+        public IActionResult ThankYouCreate(string name, int? id)
+        {
+            return View(new ThankYouCreateProjectViewModel
             {
-                Name = project.Name
+                Name = name
```

Comment:
This will show the "changed" name to the user. Probably best to just retrieve the actual name from the database.

Reply:
Well noticed, fixed in c519dac.